setup.py

```diff
@@ -56,6 +56,9 @@
     'github': [
         'github3.py>=0.9',
     ],
+    'https': [
```

Comment:
I'm not sure we should add gunicorn as a requirement here - it's a pretty big package and there's many other ways to run a WSGI app. Perhaps just add it in the documentation of the example app instead.